Assets/Tests/InputSystem.Editor/XRIPackageTest.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.TestTools;
+using UnityEditor.PackageManager;
+using UnityEditor.PackageManager.Requests;
+
+// Disable irrelevant warning about there not being underscores in method names.
+#pragma warning disable CA1707
+
+public class XRIPackageTests
+{
+	static AddRequest XRAddRequest;
+    static RemoveRequest XRRemoveRequest;
+	
+    [UnityTearDown]
+    public IEnumerator TearDown()
+    {
```

Comment:
Maybe make a comment that this is associated to the test fetching the XRI package and might need to be reworked as a TearDown function if we add additional tests that doesn't operate on the package itself

Reply:
done